environment-M1.yml

```diff
@@ -0,0 +1,34 @@
+name: ogzaf-dev
+channels:
+- conda-forge
+dependencies:
+- python>=3.7.7, <=3.9.12  # This restriction can be removed as soon as these packages support Python 3.10
+#- mkl>=2020.2  ## this fails on M1 Mac: "ResolvePackageNotFound:  - mkl[version='>=2020.2']"
```

Comment:
```suggestion
#- openssl=1.1.1 #to ensure compatibility with the UN Population API
```

Reply:
To address an incompatibility with openssl version 3
[Discussion here](https://github.com/EAPD-DRB/OG-ZAF/issues/4#issuecomment-1312568482)

Reply:
@SeaCelo. Let's update the name of the `environment-M1.yml` in line 1 to `ogzaf-M1-dev`.